public/templates/master.html

```
@@ -0,0 +1,189 @@
+            line-height: 24px;
+            margin: 16px 0;
+        }
+
+        .footer-bottom {
+            padding: 0.5rem;
+            border-bottom-right-radius: 0.5rem;
+            border-bottom-left-radius: 0.5rem;
+            max-width: 37.5em;
+        }
+    </style>
+</head>
```

HTMLHint (1.5.0)

[error] <title> must be present in <head> tag.

(title-require)